packages/@react-aria/slider/docs/useSlider.mdx

```diff
@@ -0,0 +1,413 @@
+  maxValue={1}
+  step={0.01} />
+```
+
+### Multi thumb
+
+This example shows how to build a slider with multiple thumbs. The thumb component is the same one shown in the previous
+example. The main difference in this example is that there are two `<Thumb>` elements rendered with different `index` props.
+In addition, the `<output>` element uses `state.getThumbValueLabel` for each thumb to display the selected range.
+
+```tsx example
+function RangeSlider(props) {
+  let trackRef = React.useRef(null);
+
+  let numberFormatter = useNumberFormatter(props.formatOptions);
+  let state = useSliderState({...props, numberFormatter});
+  let {
+    containerProps,
+    trackProps,
+    labelProps,
+    outputProps
+  } = useSlider(props, state, trackRef);
+
+  return (
+    <div
+      {...containerProps}
+      style={{position: 'relative', display: 'flex', flexDirection: 'column', alignItems: 'center', width: 300, touchAction: 'none'}}>
+      <div style={{display: 'flex', alignSelf: 'stretch'}}>
+        {props.label &&
+          <label {...labelProps}>{props.label}</label>
+        }
+        <output {...outputProps} style={{flex: '1 0 auto', textAlign: 'end'}}>
+          {`${state.getThumbValueLabel(0)} - ${state.getThumbValueLabel(1)}`}
+        </output>
+      </div>
+      <div
+        {...trackProps}
+        ref={trackRef}
+        style={{
+          position: 'relative',
+          height: 30,
+          width:' 100%'
+        }}>
+        <div style={{
+          position: 'absolute',
+          backgroundColor: 'grey',
+          height: 3,
+          top: 13,
+          width: '100%'
+        }} />
+        <Thumb index={0} state={state} trackRef={trackRef} />
+        <Thumb index={1} state={state} trackRef={trackRef} />
+      </div>
+    </div>
+  );
+}
+
+///- begin collapse -///
+function Thumb(props) {
+  let {state, trackRef, index} = props;
+  let inputRef = React.useRef(null);
+  let {thumbProps, inputProps} = useSliderThumb({
+    index,
+    trackRef,
+    inputRef
+  }, state);
+
+  let {focusProps, isFocusVisible} = useFocusRing();
+  return (
+    <div
+      style={{
+        position: 'absolute',
+        top: 4,
+        transform: 'translateX(-50%)',
+        left: `${state.getThumbPercent(index) * 100}%`
+      }}>
+      <div
+        {...thumbProps}
+        style={{
+          width: 20,
+          height: 20,
+          borderRadius: '50%',
+          backgroundColor: isFocusVisible
+            ? 'orange'
+            : state.isThumbDragging(index)
+              ? 'dimgrey'
+              : 'gray'
+        }}>
+        <VisuallyHidden>
+          <input ref={inputRef} {...mergeProps(inputProps, focusProps)} />
+        </VisuallyHidden>
+      </div>
+    </div>
+  );
+}
+///- end collapse -///
+
+<RangeSlider
+  label="Price Range"
+  formatOptions={{style: 'currency', currency: 'USD'}}
+  maxValue={500}
+  defaultValue={[100, 350]}
+  step={10} />
+```
+
+### Vertical orientation
```

Comment:
If we have a vertical slider in the docs, should we have it in storybook too? Github issue for this?